src/components/organisms/NewProfileModal/components/buttons/ProfileModalButtons/ProfileModalButtons.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useMemo } from "react";
+
+import {
+  profileModalWideButtonCustomStyle,
+  profileModalWideButtonCustomStyleGrey,
+} from "types/profileModal";
+import { User } from "types/User";
+import { ContainerClassName } from "types/utility";
+
+import { WithId } from "utils/id";
+
+import { useIsCurrentUser } from "hooks/useIsCurrentUser";
+import { useIsOnline } from "hooks/useIsOnline";
+
+import { Button } from "components/atoms/Button";
+
+import "./ProfileModalButtons.scss";
+
+export interface ProfileModalButtonsProps extends ContainerClassName {
+  onClick: () => void;
+  user: WithId<User>;
+}
+
+export const ProfileModalButtons: React.FC<ProfileModalButtonsProps> = ({
+  containerClassName,
+  onClick,
+  user,
+}) => {
+  const { isOnline } = useIsOnline(user.id);
+  const isCurrentUser = useIsCurrentUser(user);
+
+  const sendMessageButtonStyle = useMemo(
+    () =>
+      isCurrentUser
+        ? profileModalWideButtonCustomStyleGrey
+        : isOnline
+        ? {
+            ...profileModalWideButtonCustomStyle,
+            backgroundColor: "#78B553",
+          }
+        : profileModalWideButtonCustomStyle,
+    [isOnline, isCurrentUser]
+  );
```

Comment:
This is a legacy. Why do we need this complicated logic? Can we just use classNames ?

Reply:
`components/atoms/Button` is using JSS which embed styles in the `style` attribute of an element. That way we cannot override it with `className`
And it cannot be quickly refactored as it's used in a couple of places:
![image](https://user-images.githubusercontent.com/16973041/129356263-08164810-b093-434a-b625-c2e55a765c93.png)

I can do that in a separate PR.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1934/commits/9a89efc5877a94e36edd372aba8c28c8dc82304d